docs/procedures/general_checks.md

```diff
@@ -0,0 +1,110 @@
+# General Checks
+
+## Before Measuring
+
+Here are a few checks to perform before starting measurements.
+These are updated as much as possible for Run 3 values.
+
+- [ ] <details class="nodeco"><summary>Make Sure Intensity Is below $10^{10}$ppb until Local Corrections Are In</summary>
+      <p> After local corrections, we can move to having 3 bunches at $10^{10}$ppb evenly spaced along the ring.
```

Comment:
Every single bunch still needs to be below 10^10

Reply:
Done in [a5c6908](https://github.com/pylhc/pylhc.github.io/pull/69/commits/a5c6908372ef4055d5b99ee516d3330401ec7eae)